DFC.ServiceTaxonomy.Theme/wwwroot/Scripts/trumbowyg.gds.js

```diff
@@ -261,6 +276,48 @@
         trumbowyg.restoreRange();
     }
 
+    function youtubeLink(trumbowyg) {
+
+        var options = {
+            url: {
+                label: trumbowyg.lang.iframeSettings.url,
+                required: true
+            },
+            title: {
+                label: trumbowyg.lang.iframeSettings.title
+            },
+            height: {
+                label: trumbowyg.lang.iframeSettings.height,
+                required: true,
+                value: "360"
+            },
+            width: {
+                label: trumbowyg.lang.iframeSettings.width,
+                required: true,
+                value: "640"
+            },
+        };
+
+        trumbowyg.openModalInsert(
+            trumbowyg.lang.youtubeLink,
+            options,
+            function (data) {
+
+                var $linkToEmbed = $('<iframe>')
+                    .attr('src', data.url + '?rel=0')
```

Comment:
Does this correctly handle where the URL already has query params?

E.g. when i go to a youtube video, the url looks like this...
`https://www.youtube.com/watch?v=A3svABDnmio&list=RDA3svABDnmio&start_radio=1`

Would we end up with an url with 2 `?`'s in it?

Reply:
Update code to handle this issue.